service/Makefile

```diff
@@ -69,12 +65,6 @@ SRC_FILES := \
 	guestos.pb-c.c \
 	service.c
 
-SRC_FILES_EXEC_CAP_SYSTIME := \
```

Comment:
These changes are not related to Systemd

Reply:
made a separate fixes commit.